frontend/src/service/CocktailsApiService.ts

```diff
@@ -4,4 +4,10 @@ import axios from "axios";
 export const getAllCocktails: () => Promise<Cocktail[]> = () => {
     return axios.get<Cocktail[]>("cocktail")
```

Comment:
müsste es hier nicht 
axios.get("/cocktail")

oder so ähnlich heißen?

Reply:
Du bist die Beste . :D Mit Flo haben wir der Grund die 2* 500 Fehlercodes gesucht und haben nichts gefunden. Das war mehr als 1 Stunde. :D :D Und dann kommst du und löst  das Problem in 5mins. :D